src/tools/genpolicy/src/utils.rs

```diff
@@ -82,13 +82,23 @@ struct CommandLineOptions {
         help = "Registry that uses plain HTTP. Can be passed more than once to configure multiple insecure registries."
     )]
     insecure_registry: Vec<String>,
+
+    #[clap(
+        short,
+        long,
+        help = "If specified, resources that have a runtimeClassName field defined will only receive a policy if the parameter is a prefix of the name",
+        default_missing_value = "kata-cc", // used if flag is present but no value is given
+        num_args = 0..=1,
+    )]
+    runtime_class_name: Option<String>,
```

Comment:
@3u13r please look into supporting multiple runtime_class_names here - just in case users will need more than one in the future.